scripts/Dockerfile.devel

```diff
@@ -1,12 +1,19 @@
 FROM ceph/ceph:v15
 
+RUN \
+ mkdir -p /usr/local/go && \
+ curl https://storage.googleapis.com/golang/go1.13.9.linux-amd64.tar.gz | tar xzf - -C /usr/local/go --strip-components=1
+
 ENV GOPATH=/go \
-	GO111MODULE=on
+ GOROOT=/usr/local/go \
+ GO111MODULE=on
+
+ENV PATH=$GOROOT/bin:$GOPATH/bin;$PATH
```

Comment:
This is incorrect, not sure how this even can work... The last component in the value contains `;` and not `:`.

Also please add all environment variables together, before `RUN` and consume the variable (`GOROOT`) in the `RUN` command.

Reply:
Done.